vspace/src/main/java/edu/asu/diging/vspace/web/publicview/ExhibitionSequencesController.java

```diff
@@ -54,12 +60,20 @@ public String sequence(Model model, @PathVariable("sequenceId") String sequenceI
         if (sequenceExist==null) {
             throw new SequenceNotFoundException(sequenceId);
         }
-        
+
         List<ISlide> slides = sequenceManager.getSequence(sequenceId).getSlides();
         if (slides.size() == 0) {
             throw new SlidesInSequenceNotFoundException();
         }
+
+        if(choiceId==null) {
+            choiceId=sequenceId;
+            branchingPointId = sequenceManager.getSequence(sequenceId).getSlides().get(0).getId();
+        }
+        if(choice){
```

Comment:
just check if there is a branching point id

Reply:
this

Reply:
removing this flag, not required :-)